spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -224,6 +315,20 @@ data class SpdxLicenseWithExceptionExpression(
         } ?: throw SpdxException("'$exception' is not a valid SPDX license exception id.")
     }
 
+    override fun equals(other: Any?) =
+        when (other) {
+            is SpdxLicenseWithExceptionExpression -> license == other.license && exception == other.exception
+            is SpdxExpression -> {
```

Comment:
Nit: For consistency with the other `when`s where either case is a block we'd also need a blank line here.

Reply:
But then again, we'd also need a blank line before `else` then, and that's probably just overdoing it. Maybe we should actually remove the blank lines again in cases where the first case is a one-liner 😆

Reply:
I think we don't need to be 100% consistent with those blank lines, as long as we don't want to define a rule like "add a blank line if the block is larger than x lines".